core/tests/webdriverio_utils/ContributorDashboardPage.js

```diff
@@ -251,6 +267,23 @@ var ContributorDashboardPage = function() {
       'Submit question tab button', submitQuestionTabButton);
     await this.waitForOpportunitiesToLoad();
   };
+
+  this.navigateToMyContributionTab = async function() {
+    await action.click(
+      'Submit question tab button', myContributionTabButton);
+    await this.waitForOpportunitiesToLoad();
+  };
+
+  this.selectTranslationReviewTab = async function() {
+    var reviewTabs = await reviewTabsSelector();
```

Comment:
Do we have multiple review tabs button in view?

Reply:
Yeah, we have multiple review tabs. I think review buttons would be a more suitable name for it.